lib/livebook/config.ex

```diff
@@ -26,6 +26,14 @@ defmodule Livebook.Config do
     Application.fetch_env!(:livebook, :default_runtime)
   end
 
+  @doc """
+  Returns the enable runtime module.
+  """
+  @spec enable_runtime(Livebook.Runtime.t()) :: boolean()
+  def enable_runtime(runtime) do
```

Comment:
This would be a good function to finish with a question mark.

```suggestion
  @doc """
  Returns if the runtime module is enabled.
  """
  @spec enable_runtime?(Livebook.Runtime.t()) :: boolean()
  def enable_runtime?(runtime) do
```

Reply:
I'd suggest `runtime_enabled?` :)